src/applications/pact-act/containers/questions/agent-orange/AgentOrange-2-2-3.jsx

```diff
@@ -66,11 +66,11 @@ const mapDispatchToProps = {
 
 Orange223.propTypes = {
   formResponses: PropTypes.object.isRequired,
-  setOrange223: PropTypes.func.isRequired,
-  viewedIntroPage: PropTypes.bool.isRequired,
   router: PropTypes.shape({
     push: PropTypes.func,
-  }),
+  }).isRequired,
```

Comment:
We had some inconsistency in `router` prop type declarations. This change is in the name of consistency.